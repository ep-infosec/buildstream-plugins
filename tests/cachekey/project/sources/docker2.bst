kind: import
sources:
- kind: docker
  registry-url: upstream:images
  image: theimage
  ref: thisistheimagedigest
  track: latest
