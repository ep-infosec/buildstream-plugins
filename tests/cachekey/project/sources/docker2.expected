c3a034d31c4a1e4a4a0a34df06bb82ce7e67bb7b014a8eb634c195165fbe98e5